server/src/test/java/org/apache/druid/segment/realtime/plumber/RealtimePlumberSchoolTest.java

```diff
@@ -161,6 +162,7 @@ public void setUp() throws Exception
                     new TimestampSpec("timestamp", "auto", null),
                     new DimensionsSpec(null, null, null),
                     null,
+                    null,
```

Comment:
WOW - I didn't realize we used so many constructors! Maybe one day, I'll get around to using mockito to make this refactoring less painful in the future.